resources/pod.py

```diff
@@ -0,0 +1,158 @@
+"""
+Pod related functionalities and context info
+
+Each pod in the openshift cluster will have a corresponding pod object
+"""
+import logging
+import tempfile
+import yaml
+
+from ocs.ocp import OCP
+from ocs import defaults, kinds
+from ocsci.config import ENV_DATA
+from ocs.exceptions import CommandFailed
+
+from resources.base_resource import BaseOCSClass
+
+logger = logging.getLogger(__name__)
+
+
+class Pod(BaseOCSClass):
+    """
+    Handles per pod related context
+    """
+
+    def __init__(self, **kwargs):
+        """
+        Initializer function
+
+        kwargs:
+            Copy of ocs/defaults.py::<some pod> dictionary
+        """
+        self.pod_data = kwargs
+        super(Pod, self).__init__(
+            self.pod_data['apiVersion'], self.pod_data['kind'],
```

Comment:
__init__() with get only dict when https://github.com/red-hat-storage/ocs-ci/pull/132 will be in

Reply:
ok

Reply:
done